cml/wasm/Cargo.toml

```diff
@@ -23,4 +23,4 @@ hex = "0.4.0"
 linked-hash-map = "0.5.3"
 serde_json = "1.0.57"
 serde-wasm-bindgen = "0.4.5"
-wasm-bindgen = { version = "0.2", features=["serde-serialize"] }
\ No newline at end of file
+wasm-bindgen = { version = "0.2" }
```

Comment:
What about this version? Should we put it to `0.2.87` like all the others for consistency? Same for the multi-era crate. I think it was just on `0.2` due to copying an pasting dependencies at some point.

Reply:
I imagine every crate will resolve to the same version anyway. But probably yeah, if we pin one it's probably best to pin all of them.

Reply:
Although I'm still not really sure the pin is needed